students/SLammertink/Lesson03/slicing_lab.py

```diff
@@ -0,0 +1,45 @@
+# Author: SLammertink
+# Lesson 3 Slicing Lab
+
+# Function with the first and last items exchanged.
+
+def first_last(x):
+    return x[-1:] + x[1:-1] + x[:1]
+
+
+# Function with every other item removed.
+
+def rem_every_other(y):
+    return y[::2]
+
+# Function with the first 4 and the last 4 items removed, and then every other item in between.
+
+def rem_first_last_every_other(z):
+    return z[4:-4:2]
+
+# Function with the elements reversed (just with slicing).
+
+def reversed(a):
```

Comment:
be careful about over-writing built-in names; you should choose a name that does not conflict.